python/tvm/relay/op/_tensor.py

```diff
@@ -196,6 +200,9 @@ def elemwise_shape_func(attrs, inputs, _):
 register_shape_func("less_equal", False, broadcast_shape_func)
 register_shape_func("greater", False, broadcast_shape_func)
 register_shape_func("greater_equal", False, broadcast_shape_func)
+register_shape_func("maximum", False, broadcast_shape_func)
```

Comment:
add test case for maximum/mininum/exp

Reply:
broadcast has already been covered. Currently in test_any we don't test every op in the same category. I add test any for elemwise